nestkernel/connector_base.h

```diff
@@ -98,6 +129,7 @@ class ConnectorBase
 
   virtual size_t get_num_connections() = 0;
   virtual size_t get_num_connections( synindex syn_id ) = 0;
+  virtual size_t get_num_connections( size_t target_gid, size_t thrd, synindex syn_id ) = 0;
```

Comment:
What is the reason for introducing the third `get_num_connections` method? As far as I can see, it is called only in the `HetConnector::get_num_connections` version of this method and nowhere else... I am confused :confused: 

The same goes for the `get_target_gids` method.